### DISCUSSION TITLE
Reimport a renamed tcx file does not change the importfilename in my tourbook anymore

### DISCUSSION THREAD

**Discussion by evdhoek** [mytourbook/mytourbook#1170] [Q&A]
In the previous version before 23.8.0, after reimport a renamed tcx file, a helpscreen was shown with 4 options to adapt the new name into the database.
I do not know wheater it is has to do with the recent update, but now I do not see the helpscreen anymore and the importfilename in my tourbook is not changed anymore.
Old name already in MyTourbook: 20230617_1030 f Enschede-TechnBase+ 35.14
Changed into: 20230617_1030 f Enschede, NS-TechnBase+ 35.14
I hope someone can help.

After import I read In Tourlog: 
15:41:21 [main - 1] 1  Importing tours... 
15:41:21 [Importing tours - 54] 2 [TCX] C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230617_1030 f Enschede, NS-TechnBase+ 35.14.tcx - is empty
15:41:21 [Importing tours - 54] 3 0.531 s ∙ 2,474 # ∙ 17/06/2023, 10:31 ← C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230617_1030 f Enschede, NS-TechnBase+ 35.14.tcx
15:41:21 [Loading imported tours - 55] 4 [Skipped import file] C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230617_1030 f Enschede, NS-TechnBase+ 35.14.tcx in tour 17/06/2023, 10:31
15:41:22 [main - 1] 5 Imported in 1.629 s

**Comment by wolfgang-ch** [comment]
The re-import is terminated because of

15:41:21 [Importing tours - 54] 2 [TCX] C:\Users\elevd\OneDrive\Documents.Garmin.Import\20230617_1030 f Enschede, NS-TechnBase+ 35.14.tcx - **is empty**

https://github.com/mytourbook/mytourbook/blob/d520c1f1a0a3f0f75f84582ec8475cc651388e45/bundles/net.tourbook.device.garmin/src/net/tourbook/device/garmin/GarminTCX_SAXHandler.java#L632-L640

**Reply by evdhoek** [reply]
Hello,
Thanks for your answer. Sorry to boher you once more.
To reproduce I did a test with another file _**20230818_1132 a This is a Test 30.57.gpx**_ and renamed it to 20230818_1132 a Test 30.57.gpx
I got the mesage: skipped import file and did not get the menu with the 4 options for keep the filename in the database or change the filename.

Tourlog:
12:07:10 [main - 1                 ] 1              Importing tours...
12:07:10 [Importing tours - 54     ] 2     OK          0.039 s ∙  1,012 # ∙ 18/08/2023, 11:32 ← C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230818_1132 a This is a Test 30.57.gpx
12:07:10 [main - 1                 ] 3              Imported in 0.324 s
12:07:14 [main - 1                 ] 4              Save tours
12:07:14 [main - 1                 ] 5                 Saved 1 tours in 0.124 s
12:08:06 [main - 1                 ] 6              Importing tours...
12:08:06 [main - 1                 ] 7              Imported in 0.334 s
12:08:45 [main - 1                 ] 8              Importing tours...
12:08:45 [Importing tours - 61     ] 9     OK          0.029 s ∙  1,012 # ∙ 18/08/2023, 11:32 ← C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230818_1132 a Test 30.57.gpx
12:08:45 [Loading imported tours - 62] 10    INFO        [Skipped import file] C:\Users\elevd\OneDrive\Documents\.Garmin\.Import\20230818_1132 a Test 30.57.gpx in tour 18/08/2023, 11:32
12:08:45 [main - 1                 ] 11             Imported in 0.298 s

**Comment by wolfgang-ch** [comment]
Created issue https://github.com/mytourbook/mytourbook/issues/1171
